src/operator/nn/mkldnn/mkldnn_base.cc

```diff
@@ -128,8 +128,14 @@ void CommitOutput(const NDArray &arr, const mkldnn_output_t &res) {
   if (res.first == CopyBack) {
     const_cast<NDArray &>(arr).CopyFrom(*res.second);
   } else if (res.first == AddBack) {
-    auto mem = arr.GetMKLDNNData(res.second->get_primitive_desc());
-    CHECK(mem != nullptr);
```

Comment:
I wonder whether it works if we just use GetMKLDNNDataReorder?

Reply:
I think it can work but it will be a little confusion to call reorder API for all cases.